Per https://github.com/rails/rails/pull/46735#issuecomment-1352643705, this
is just dropping a test which doesn't make sense for ruby32.

https://github.com/rails/rails/issues/46712
https://github.com/rails/rails/pull/46735

From 60ffaac2e9644076e53afa1a2b1a716e289b7085 Mon Sep 17 00:00:00 2001
From: Akira Matsuda <ronnie@dio.jp>
Date: Thu, 15 Dec 2022 15:45:27 +0900
Subject: [PATCH] RubyVM class serial is no longer available in Ruby 3.2

since ruby/ruby@13bd617ea6fdf72467c593639cf33312a06c330c
--- a/test/executor_test.rb
+++ b/test/executor_test.rb
@@ -226,7 +226,7 @@ def test_hook_insertion_order
   end
 
   def test_class_serial_is_unaffected
-    skip if !defined?(RubyVM)
+    skip if !defined?(RubyVM) || !RubyVM.stat.has_key?(:class_serial)
 
     hook = Class.new do
       define_method(:run) do
